### PR TITLE
Group B Vallie

### PR DESCRIPTION
**Pull request by vmjoseph**

### CODE REVIEW

**Comment by JasonCorriveau** [comment]
Awesome Vallie, love the GIF!

**Review by JasonCorriveau — APPROVED** [review]

**Comment by zchilds** [comment]
Can you 3D print another 3D printer?

File: _posts/: 2020-06-05-Vallie-Joseph.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "A new hobby or skill I've picked up during lockdown..."
+author: Vallie
+date: 2020-06-03
+---
+
+![Image of 3D Printer](https://media.giphy.com/media/R3H7UiNMdykYE/giphy.gif)
+
+Comment your reply here.
```
Comment (qianshenghu) [inline _posts/: 2020-06-05-Vallie-Joseph.md]: Q replied "Can I bother you to make me stuff? :)"

File: _posts/: 2020-06-05-Vallie-Joseph.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: "A new hobby or skill I've picked up during lockdown..."
+author: Vallie
+date: 2020-06-03
+---
+
+![Image of 3D Printer](https://media.giphy.com/media/R3H7UiNMdykYE/giphy.gif)
```
Comment (zach004) [inline _posts/: 2020-06-05-Vallie-Joseph.md:7]: Can you 3D Print another 3D Printer?
-ZachC

File: _posts/: 2020-06-05-Vallie-Joseph.md (hunk 1 above)
Comment (vmjoseph) [reply _posts/: 2020-06-05-Vallie-Joseph.md]: $$

File: _posts/: 2020-06-05-Vallie-Joseph.md (hunk 2 above)
Comment (vmjoseph) [reply _posts/: 2020-06-05-Vallie-Joseph.md:7]: We'll find out!

File: _posts/: 2020-06-05-Vallie-Joseph.md (hunk 1 above)
Comment (KCoterwas) [inline _posts/: 2020-06-05-Vallie-Joseph.md]:
```suggestion
Kim replied: That Isabelle print you made for me is the best :)
```

Comment (alliyahtaylor) [inline _posts/: 2020-06-05-Vallie-Joseph.md]:
```suggestion
Waiting for more buff pikachus
```

Comment (vmjoseph) [reply _posts/: 2020-06-05-Vallie-Joseph.md]: they are on their way

**Pull request merged**